frontend/src/components/pages/CampsList/CampsTable.tsx

```diff
@@ -215,6 +225,16 @@ const CampsTable = ({
                         Delete camp
```

Comment:
I think delete belongs in the popover footer

Reply:
pretty sure delete was in the popover too? This was unchanged from before though so I am not 100% sure